.github/workflows/single-card-demo-tests-impl.yaml

```
@@ -12,38 +12,60 @@ on:
+          { name: "qwen7b", runner-label: "N300", performance: false, cmd: run_qwen7b_func, owner_id: U06F3ER8X9A}, # Stuti Raizada
+          { name: "llama3", runner-label: "N300", performance: false, cmd: run_llama3_func, owner_id: U03PUAKE719}, # Miguel Tairum
+          { name: "vgg", runner-label: "N300", performance: false, cmd: run_vgg_func, owner_id: U06Q7ESTFEV}, # Borys Bradel
+          { name: "bert_tiny", runner-label: "N300", performance: false, cmd: run_bert_tiny_func, owner_id: U024A4EFV6U}, # Brian Liu
+          { name: "bert", runner-label: "N300", performance: false, cmd: run_bert_func, owner_id: U024A4EFV6U}, # Brian Liu
+          { name: "resnet", runner-label: "N300", performance: false, cmd: run_resnet_func, owner_id: U0837MYG788}, # Marko Radosavljevic
+          { name: "distilbert", runner-label: "N300", performance: false, cmd: run_distilbert_func, owner_id: U013121KDH9}, # Austin Ho
+          { name: "covnet_mnist", runner-label: "N300", performance: false, cmd: run_covnet_mnist_func, owner_id: U06LRK6JDGB}, # Saad Jameel
+          { name: "mnist", runner-label: "N300", performance: false, cmd: run_mnist_func, owner_id: U06ECNVR0EN}, # Evan Smal
+          { name: "squeezebert", runner-label: "N300", performance: false, cmd: run_squeezebert_func, owner_id: UBHPP2NDP}, #Joseph Chu
+          { name: "roberta", runner-label: "N300", performance: false, cmd: run_roberta_func, owner_id: U06F3ER8X9A}, # Stuti Raizada
+          { name: "mistral7b", runner-label: "N150", performance: true, cmd: run_mistral7b_perf, owner_id: U03PUAKE719}, # Miguel Tairum
+          { name: "mistral7b", runner-label: "N300", performance: true, cmd: run_mistral7b_perf, owner_id: U03PUAKE719}, # Miguel Tairum
+          { name: "llama3", runner-label: "N300", performance: true, cmd: run_llama3_perf, owner_id: U03PUAKE719}, # Miguel Tairum
+          { name: "falcon7b", runner-label: "N300", performance: true, cmd: run_falcon7b_perf, owner_id: U05RWH3QUPM}, # Salar Hosseini
+          { name: "mamba", runner-label: "N300", performance: true, cmd: run_mamba_perf, owner_id: U06ECNVR0EN}, # Evan Smal
+          { name: "whisper", runner-label: "N300", performance: true, cmd: run_whisper_perf, owner_id: U05RWH3QUPM}, # Salar Hosseini
         ]
-    name: ${{ matrix.test-group.name }}
+    name: ${{ matrix.test-group.name }}-${{ matrix.test-group.runner-label }}-${{ matrix.test-group.performance && 'perf' || 'func' }}
     env:
-      ARCH_NAME: ${{ matrix.test-group.arch }}
+      #ARCH_NAME: ${{ inputs.arch }}
       LOGURU_LEVEL: INFO
       LD_LIBRARY_PATH: ${{ github.workspace }}/build/lib
-    runs-on: ${{ matrix.test-group.runs-on }}
+    runs-on: 
+      - ${{ matrix.test-group.runner-label }}
+      - in-service
+      - ${{ matrix.test-group.runner-label == 'N300' && matrix.test-group.performance && 'bare-metal' || 'cloud-virtual-machine' }}
```

Right now it doesn't matter since we're not attempting to run perf on anything except N300, so why limit it?

Conceptually we would want a BM for perf even for a N150 or P300 card.  We would need to remember to come in and change this or we'd get the wrong machine.
Leaving it out, we'll be alerted to the issue when no machine picks up the job.

```suggestion
      - ${{ matrix.test-group.performance && 'bare-metal' || 'cloud-virtual-machine' }}
```